v1/pipeline/steps/custom/key-values-criteria-spec.yaml

```diff
@@ -9,11 +9,8 @@ allOf:
       type: array
```

Comment:
is `$ref: criteria-spec.yaml` required in V1, since we have made changes in all approvals

Reply:
It is being used in key-values-criteria-spec.yaml, we have not deleted both the classes for now as we might need to get the key-values back

Reply:
yeah I meant we can remove the usage from key-values-criteria-spec.yaml, since anyways jexl spec has been deleted